api/apps/api/src/modules/geo-features/geo-feature.api.entity.ts

```diff
@@ -94,6 +94,10 @@ export class GeoFeature extends BaseEntity {
     referencedColumnName: 'id',
   })
   createdBy?: User;
+
+  @ApiPropertyOptional()
+  @Column('boolean', { name: 'is_custom' })
+  isCustom?: boolean;
```

Comment:
do we need to add this to `get GeoFeaturesService.serializerConfig()`?

honestly, I don't remember - I haven't added new fields and exposed them through the JSON:API payloads for quite some time now 😬

Reply:
yeah, I think we do, and I think that's why E2E tests are failing! Thanks for pointing out, I'm force pushing an updated commit!